Package.swift

```diff
@@ -46,32 +46,41 @@ let package = Package(
             name: "AzureCore",
             dependencies: [],
             path: "sdk/core/AzureCore",
+            exclude: ["Source/Supporting Files"],
             sources: ["Source"]
         ),
         .target(
             name: "AzureCommunication",
             dependencies: ["AzureCore"],
             path: "sdk/communication/AzureCommunication",
+            exclude: ["Source/Supporting Files"],
             sources: ["Source"]
         ),
         .target(
             name: "AzureCommunicationChat",
             dependencies: ["AzureCore"],
             path: "sdk/communication/AzureCommunicationChat",
+            exclude: ["Source/Supporting Files"],
             sources: ["Source"]
         ),
         // Test targets
         .testTarget(
             name: "AzureCoreTests",
             dependencies: ["AzureCore"],
             path: "sdk/core/AzureCore",
+            exclude: [
+              "Tests/Info.plist",
+              "Tests/Data Files",
+              "Tests/test.playground"
```

Comment:
Why do we have a `test.playground`?

Reply:
That's a great question, I don't know :p When I updated to the Swift 5.3 toolchain, `swift build` started failing because it doesn't seem to ignore as many file types by default as the older toolchain, and these were files I needed to exclude to get the build working again. If we don't need it we could just delete the playground.

Reply:
Let's delete the playground. We could also add that to the gitignore.